gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 20 10:23:34 CET 2017
+#Mon Apr 01 12:15:24 CEST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-all.zip
```

Comment:
What was this needed for?

Reply:
I am not sure.. Gradle did this when I pulled in the new drivers I think..